libs/books/feature/src/lib/book-search/books-grid/books-grid-item/books-grid-item.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="book--title">
+  {{ book.title }}
+</div>
+<div class="book--content">
+  <div class="book--content--cover">
+    <img src="{{ book.coverUrl }}" alt="{{ 'cover of ' + book.title}}" />
```

Comment:
This isn't really necessary, and is probably just annoying for screen reader users.

cf. https://www.w3.org/WAI/tutorials/images/decision-tree/